eslint.config.mjs

```diff
@@ -832,6 +850,12 @@ export default typescript.config([
               group: ['sentry/components/devtoolbar/*'],
               message: 'Do not depend on toolbar internals',
             },
+            {
+              group: ['sentry/utils/theme*', 'sentry/utils/theme'],
+              importNames: ['lightTheme', 'darkTheme', 'default'],
+              message:
+                "Use 'useTheme' hook of withTheme HOC instead of importing theme directly. For tests, use ThemeFixture.",
+            },
```

Comment:
I think this is the first universal restricted pattern, so could be time to make a `restrictedImportPatterns` variable. The other patterns are more specific to each folder where the rule applies, so i didn't make that var until now. 

NBD either way.